crates/icrate/src/lib.rs

```diff
@@ -42,5 +42,9 @@ pub mod CoreAnimation;
 pub mod CoreData;
 #[cfg(feature = "Foundation")]
 pub mod Foundation;
+#[cfg(feature = "JavaScriptCore")]
+pub mod JavaScriptCore;
 #[cfg(feature = "Metal")]
 pub mod Metal;
+#[cfg(feature = "WebKit")]
+pub mod WebKit;
```

Comment:
Also add an entry in `header-translator/src/data/...`, to ease the process of making methods in `WebKit` safe